lucille-core/src/main/java/com/kmwllc/lucille/core/PublisherImpl.java

```diff
@@ -122,6 +125,7 @@ private void publishInternal(Document document) throws Exception {
     }
 
     if (previousDoc.getId().equals(document.getId())) {
+      log.debug("Sending document.");
```

Comment:
this clause isn't when we're sending the document
this clause is when we're "collapsing" the current document into the previous one because they had the same IDs